courses/machine_learning/deepdive2/introduction_to_tensorflow/solutions/3_keras_sequential_api.ipynb

```diff
@@ -824,15 +824,15 @@
      "output_type": "stream",
      "text": [
       "Using endpoint [https://ml.googleapis.com/]\n",
-"OUTPUT_1\n",
-"[0.04512186720967293]\n"
+      "OUTPUT_1\n",
+      "[0.04512186720967293]\n"
      ]
     }
    ],
    "source": [
     "# The `gcloud ai-platform predict` sends a prediction request to AI Platform for the given instances.\n",
     "# TODO 5b\n",
-    "!gcloud ai-platform predict --model taxifare --json-instances input.json --version dnn"
+    "!gcloud ai-platform predict --model taxifare --json-instances input.json --version dnn --region us-east1"
```

Comment:
Maybe the region here could also be a parameter as `%REGION` -- but it is also fine as it is now, as otherwise it could not be a one-liner. (it would turn into the one below:)
```suggestion
    "%%bash\n",
    "REGION=us-east1\n",
    "!gcloud ai-platform predict --model taxifare --json-instances input.json --version dnn --region $REGION"
```